src/main/java/teammates/logic/api/Logic.java

```diff
@@ -785,16 +786,20 @@ public static boolean isCourseArchived(CourseAttributes course, InstructorAttrib
     /**
      * Preconditions: <br>
      * * All parameters are non-null.
+     * <br>
+     * The second parameter ctaMap is used to keep the query results in the form of
```

Comment:
Use the usual javadoc format for parameter description. No need to say `The second parameter...`